addons/go.mod

```diff
@@ -24,3 +24,60 @@ require (
 	sigs.k8s.io/cluster-api v1.0.1
 	sigs.k8s.io/controller-runtime v0.10.3
 )
+
+require (
```

Comment:
where do these new `require` blocks come from? are they necessary?

Reply:
These are due to the new [module graph changes](https://go.dev/doc/go1.17#tools) in go 1.17. This is the result of running `go mod tidy -go=1.17` in each module.